src/Umbraco.Infrastructure/Logging/Viewer/SerilogLogViewerSourceBase.cs

```diff
@@ -10,11 +11,13 @@ namespace Umbraco.Cms.Core.Logging.Viewer
     public abstract class SerilogLogViewerSourceBase : ILogViewer
     {
         private readonly ILogViewerConfig _logViewerConfig;
+        private readonly ILogLevelLoader _logLevelLoader;
         private readonly global::Serilog.ILogger _serilogLog;
 
-        protected SerilogLogViewerSourceBase(ILogViewerConfig logViewerConfig, global::Serilog.ILogger serilogLog)
+        protected SerilogLogViewerSourceBase(ILogViewerConfig logViewerConfig, ILogLevelLoader logLevelLoader, global::Serilog.ILogger serilogLog)
         {
             _logViewerConfig = logViewerConfig;
+            _logLevelLoader = logLevelLoader;
             _serilogLog = serilogLog;
         }
```

Comment:
I believe that this would count as a breaking change, if someone has overwritten the `ILogViewer` by inheriting from this class their code will break because the constructor have changed. 

Instead we could still add the new constructor, but keep the old one, obsolete it, and then get the service with the static service locator, something like this: 

```C#
protected SerilogLogViewerSourceBase(ILogViewerConfig logViewerConfig, global::Serilog.ILogger serilogLog)
{
    _logViewerConfig = logViewerConfig;
    _logLevelLoader = StaticServiceProvider.Instance.GetRequiredService<ILogLevelLoader>();
    _serilogLog = serilogLog;
}
```

Reply:
That's a valid point, I will fix up 😊